app/controllers/order_detail_stored_files_controller.rb

```diff
@@ -81,4 +83,18 @@ def ability_resource
     @order_detail
   end
 
+  def redirect_to_facility_order(consider_merge_order: false)
```

Comment:
If you can add some code comments here about why this logic needs to be this way that would be helpful for future developers.